CONTRIBUTING.md

```diff
@@ -177,6 +177,8 @@ Here are the trigger phrases for individual checks:
 * `/test-ipv6-only-networkpolicy`: Linux IPv6 only networkpolicy tests
 * `/test-flexible-ipam-e2e`: Flexible IPAM e2e tests
 * `/test-multicast-e2e`: Multicast e2e tests
+* `/test-multicluster-e2e`: Multicluster e2e tests
+* `/test-vm-e2e`: ExternalNode e2e tests
```

Comment:
/test-whole-conformance
/test-hw-offload

Reply:
OK, Can you help to clarify the comment of the trigger?
```
/test-whole-conformance  All conformance tests on Linux.
/test-hw-offload Hardware offloading e2e tests.
```

Reply:
They are correct

Reply:
Updated.